Update affiliated package review process to delegate editorial work (#90)
astropy/project: affiliated/affiliated_package_review_process.md

## Reviewing affiliated packages

*These instructions are for editors and reviewers*

The review process will be overseen and coordinated by individuals that are in
the *Affiliated package review editor* role (hereafter the *editors*). The
editors will start off by finding someone to carry out the main review of the
package (hereafter the *reviewer*). In this sense the role of the editor is
similar to that of a journal editor (and hence the role name). However this is
only partially true, because the editor may add their views to the review,
unlike a journal editor who is often strictly impartial on the review itself.
The pool of available reviewers will be anyone who has an official role on [the
Astropy team](http://www.astropy.org/team.html) or anyone who has participated
in the project enough to be familiar with the project guidelines and requirements.
In some cases, it may be necessary to have multiple reviewers depending on the
expertise required, and if scientific or algorithmic expertise is needed which

> is not available in the project, reviewers can be requested from outside the

Author: editors, fixed

project.

> The editor sends out an email to possible reviewers one at a time to find

Author: The one at a time meant that if you need e.g. one reviewer, not to email five and then pick the first to reply. But that's perhaps too much detail. I've changed this to 'individually' to make it clear not to send blanket emails to 5 people at the same time.

How long the editors wait for reviewers to reply is up to them.

The reviewers are anonymous unless they say they are happy to not be. However, I think @eteq and I discussed this and thought it would make sense to have it the other way around - not anonymous unless requested by the reviewer. I will make that update in a separate PR though.

someone who is willing to review the package, using the template at the bottom
of this document. They should let them know at this point that the editor may
also be doing a review and synthesizing the results into a single review. Once a
reviewer accepts, the editor sends them an email with detailed review
instructions, using the template at the bottom of this document. The editor then
leaves a message on the pull request proposing the affiliated package saying:

*Thank you for proposing this package as an affiliated package! I'm happy to
confirm that your package is now under review and we'll post the results of
the review here and on the mailing list.*

Once the reviewer has finished, they send their review back by email to the
editor, who then does their own review if needed and then writes up a review
using the review template which includes a table with badges (template at the

> bottom of this document). At this point, the editors make a decision to access

Reviewer: Typo:
```suggestion
bottom of this document). At this point, the editors make a decision to accept
```

Maybe link to https://github.com/astropy/project/blob/master/affiliated/affiliated_package_review_guidelines.md in this section too.

Author: Done

the *Affiliated package review editor* role. This role
may be filled by one or more individuals. For the rest of this section, we
use the singular term *editor*, although multiple people may share this role
(and choose how to do this). The
editor will start off by finding someone to carry out the main review of the
package (hereafter the *reviewer*). In this sense the role of the editor is
similar to that of a journal editor (and hence the role name). However this is
only partially true, because here the editor may add their views to the review,
unlike a journal editor who is often strictly impartial on the review itself.
The pool of available reviewers will be anyone who has an official role on [the
Astropy team](http://www.astropy.org/team.html) or anyone who has participated
in the Astropy project enough to be familiar with the project guidelines and requirements.
In some cases, it may be necessary to have multiple reviewers depending on the
expertise required, and if scientific or algorithmic expertise is needed which
is not available in the project, reviewers can be requested from outside the
project at the editor's discretion.

> The editor sends out an email to possible reviewers, individually, to find

Reviewer: Hmm, "individually" doesn't really explain to me that you disallow blanket emails to 5 different reviewers when you only will pick one of them. I think this expectation should be its own full sentence.

Author: Does the latest commit clarify things? If not, can you make a suggestion?

## Reviewing affiliated packages

*These instructions are for editors and reviewers.*

> The review process will be overseen and coordinated by individuals that are in

Reviewer 2: ```suggestion
The review process will be overseen and coordinated by the individuals that are in
```

the *Affiliated package review editor* role. This role
may be filled by one or more individuals. For the rest of this section, we
use the singular term *editor*, although multiple people may share this role

> (and choose how to do this). The

Reviewer 2: ```suggestion
(and it is at their discretion to decide how to divide up work). The
```

editor will start off by finding someone to carry out the main review of the
package (hereafter the *reviewer*). In this sense the role of the editor is

> similar to that of a journal editor (and hence the role name). However this is
> only partially true, because here the editor may add their views to the review,

Reviewer 2: ```suggestion
similar to that of a journal editor (and hence the role name). However this analogy
is only partially accurate, because here the editor may add their views to the review,
```

unlike a journal editor who is often strictly impartial on the review itself.
The pool of available reviewers will be anyone who has an official role on [the
Astropy team](http://www.astropy.org/team.html) or anyone who has participated
in the Astropy project enough to be familiar with the project guidelines and requirements.
In some cases, it may be necessary to have multiple reviewers depending on the
expertise required, and if scientific or algorithmic expertise is needed which
is not available in the project, reviewers can be requested from outside the
project at the editor's discretion.

The editor sends out an email to possible reviewers to find someone who is
willing to review the package, using the template at the bottom of this
document. Emails should only be sent to the next potential reviewer(s) - not

> many possible reviewers at the same time. The editor should let the potential

Reviewer: Something like this would be clearer for me but I don't know if that is too much. Also, you would need to break that long sentence up if you decide to adapt it here. Thanks!

```suggestion
many possible reviewers at the same time; For instance, if the review needs one reviewer, a request should first be sent to Reviewer A only and then to Reviewer B after an X amount of time has elapsed without any reply from Reviewer A (recommended X is 4 weeks and an email telling Reviewer A that their service is no longer needed should also be sent out too). The editor should let the potential
```

Author: Thanks! I think this is getting too specific though - I think the editors should decide on such policies internally. Also in the above scenario I think you'd want to sent reminders to reviewers etc and that really is too much detail here. My point here originally was more that the editor shouldn't just email all core maintainers asking for a volunteer, but contact people individually.

reviewer know at this point that the editor may also be doing a review and
synthesizing the results into a single review. Once a reviewer accepts, the

> editor send them an email with detailed review instructions (which includes a

Reviewer 2: ```suggestion
editor sends them an email with detailed review instructions (which includes a
```

>   https://github.com/astropy/astropy-procedures/blob/master/documents/affiliated_package_review_guidelines.md

Reviewer: ```suggestion
  https://github.com/astropy/project/blob/master/affiliated/affiliated_package_review_guidelines.md
```
